pages/docs/support/permissions.mdx

```diff
@@ -1,7 +1,6 @@
 ---
```

Comment:
I don't think `docs/support/permissions` is just for enterprise users.  There are still teams and teamMembers in the community edition, and API keys.   It's OK to leave in the support section, but I'd remove the "Enterprise" title-tag